earth_enterprise/src/earthplugin/SConscript

```diff
@@ -20,47 +20,18 @@ import os
 
 Import('env')
 
-# Get Fusion Version (use same version for server).
-# Based on Andrei's code in fusion/portableglobe/SConscript.
-def VersionNumber(source):
-  """Reads the Fusion Version number."""
-  try:
-    fp = open(source, 'r')
-    line = fp.readline()
-    while len(line) != 0:
-      if line[0] != '#':
-        version = line.strip()
-        if len(version):
-          return version
-      line = fp.readline()
-    print 'Unable to find Fusion Version Number.'
-    Exit(1)
-  except Exception, exp:
-    print 'Unable to find Fusion Version Number: ', exp
-
-def WriteBuildInfoFile(build_info, version_file_path):
-  try:
-    fp = open(version_file_path, 'w')
-    fp.write(build_info)
-    fp.close()
-  except Exception, exp:
-    print 'Unable to write Build Info File: ', exp
-
-fusion_version_basic_file_path = File('#fusion_version.txt').abspath
 server_version_file_dir = Dir('.').abspath
 
 # Create 'earthplugin' directory if it doesn't exist.
 if not os.path.exists(server_version_file_dir):
   os.mkdir(server_version_file_dir)
 
-server_version_file_path = File('%s/%s' %(server_version_file_dir, 'version.txt')).abspath
-version_basic = VersionNumber(fusion_version_basic_file_path)
-build_info = ('%s-%s' % (version_basic, time.strftime('%Y%m%d')))
-WriteBuildInfoFile(build_info, server_version_file_path)
-
 # Copy files from '../earth_enterprise/src/earthplugin/..' directory.
 env.installDirExcluding('earth', Dir('#earthplugin').abspath, ['SConscript'])
 
 # Copy files from '../earth_enterprise/src/{REL_XXX|OPT_XXX}/earthplugin/..'
-# directory. There is only one file 'version.txt' in this folder.
-env.installDirExcluding('earth', '.', [])
+# directory.
+env.install('earth', File('../builddate.txt').abspath)
```

Comment:
Do we need 3 files to store a version?  Are the places that read it unable to extract specific information from one file?